deploy.desc

```diff
@@ -7,4 +7,4 @@ SECRETS=(SPLITIO_API_KEY SPLITIO_EXT_API_KEY)
 CPU_REQUEST="5m"
 MEMORY_REQUEST="85Mi"
```

Comment:
I'd probably bump the memory for the main apps up to 100Mi, they are riding really close to their 85mibs currently